### ISSUE TITLE
about dataset

### ISSUE THREAD

**Issue by pipijiev12** [MrYANG23/Audio2BS-AIWIN2022-16th#1]
In addition to the aiwin2022 data set, are there other data sets that can be trained? ?

**Comment by MrYANG23** [comment]
Yes, this code can also train on other datasets and achieved a top 7 result in this competition (https://aicontest.msxf.com/vision)

**Comment by pipijiev12** [comment]
> Yes, this code can also train on other datasets and achieved a top 7 result in this competition (https://aicontest.msxf.com/vision)
It's a pity that the competition has ended. I can't apply to obtain the data. Or can you provide relevant software for me to produce the data?

**Comment by MrYANG23** [comment]
I can't share this dataset, but any dataset in the form of speech-to-blendshape coefficient data can be used to train this model

**Comment by pipijiev12** [comment]
> I can't share this dataset, but any dataset in the form of speech-to-blendshape coefficient data can be used to train this model
Do I only need the following two types of files: data preprocessing and pretraining model?
[a2f_export_bsweight_chinese_voice_female_p01_neutral.json](https://github.com/MrYANG23/Audio2BS-AIWIN2022-16th/files/13833441/a2f_export_bsweight_chinese_voice_female_p01_neutral.json)

**Comment by pipijiev12** [comment]
> I can't share this dataset, but any dataset in the form of speech-to-blendshape coefficient data can be used to train this model
Do I only need the following two types of files: data preprocessing and pretraining model?
One is a wav file and the other is a json file containing bone parameters.

[a2f_export_bsweight_chinese_voice_female_p01_neutral.json](https://github.com/MrYANG23/Audio2BS-AIWIN2022-16th/files/13833447/a2f_export_bsweight_chinese_voice_female_p01_neutral.json)

**Comment by yonzhao** [comment]
Hi MrYang

I 'am so sorry to bother you. I have three question to ask you.
    1. Could you describe the contest data,   one person voice for 30 mins?  
    2. I trained my model, it's bad for the close sound such as 'b p m f ', do you meet the issue, or could you give me some 
    solution?
    4. For the different voice, your model's generalization is good ?
I'am looking forward to your reply.
